lib/utils.ts

```diff
@@ -239,3 +239,9 @@ export function openWindowWithUrl(url: string): void {
 
   windowOpen(url, windowConfig);
 }
+
+export function formatMkrAmount(mkrAmount: number | string): string {
```

Comment:
We have formatValue and formatMKRAmount and i'm starting to be confused, shouldn't we just use one function to format numbers through the app? 

In case we need 2 i asume is because this one rounds "near zero" values to ≈0.00 , shouldn't that be a parameter of the other format function? idk

Reply:
You're right, I didn't realize `formatValue` handled the `>999` logic as well. So we can refactor to do the `<0.01` logic too.